### PR TITLE
feat: migrate create command to standalone package

### LINKED ISSUE #24: 提取 create-tnf package 到根目录
## Background

create 命令随着 tnf 主包走会让 npm create tnf 变慢，这没有必要。

## Solution

1、根目录创建 create-tnf 目录，这是一个 npm 包，name 为 create-tnf
2、pnpm-workspace.yaml 里需要包含这个目录
3、迁移 src/create.ts 到 create-tnf 
4、create-tnf 里估计需要配套的 tsconfig.json 和 .fatherrc.ts

### PR DESCRIPTION
**Pull request by QDyanbing**
主要更改：
功能迁移：将 create 命令逻辑从主包中提取到 create-tnf 独立包。
路径调整：在主包中修改了导入路径，使其指向构建后文件的正确位置。

Close https://github.com/umijs/tnf/issues/24

---

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:

- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC.
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests

- [x] Run the tests and other checks with `pnpm ci`

### Changesets

- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features and fix bugs should all be `patch` before we release `0.1.0`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### Edits

- [x] Please ensure that 'Allow edits from maintainers' is checked. PRs without this option may be closed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "compression": "^1.7.5",
     "connect-history-api-fallback": "^2.0.0",
     "cors": "^2.8.5",
+    "create-tnf": "^0.1.2",
```
Comment (sorrycc) [inline package.json]: 抱歉，可能前面没说清楚。

1、create-tnf 单独成包，不作为主包的依赖
2、主包里删除 create 命令

File: package.json [hunk 2]
```diff
@@ -76,9 +79,7 @@
     "tsx": "^4.19.2",
     "typescript": "^5.6.3",
     "vitest": "^2.1.4",
-    "zx": "^8.2.0",
-    "@commitlint/cli": "^19.0.3",
-    "@commitlint/config-conventional": "^19.0.3"
```
Comment (sorrycc) [inline package.json]: 这个不要挪位置。

File: create-tnf/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "create-tnf",
+  "version": "0.1.2",
+  "description": "A standalone create command for TNF.",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
```
Comment (sorrycc) [inline create-tnf/package.json]: 1、应该没有 dist/types/index.d.ts 吧
2、要加个 create-tnf 的 bin
3、templates 目录要挪到 create-tnf 下

**Review by sorrycc — CHANGES_REQUESTED** [review]

**Comment by sorrycc** [comment]
记得提交前先执行 pnpm ci 。

File: package.json (hunk 1 above)
Comment (QDyanbing) [reply package.json]: 就是整个主包的create 命令提出成为单独的包；主包不再提供create相关的命令了是吧；那了解了；明天重新处理下

File: create-tnf/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  assert(cmd, 'Command is required');
+
+  return create({
+    cwd: cwd,
+    name: argv._[1] as string | undefined,
+    template: argv.template,
+  });
+}
+
+setNoDeprecation();
```
Comment (xiaohuoni) [inline create-tnf/src/index.ts]: 这些操作看起来没什么必要，感觉可以删掉 fishkit/node 和 constants 文件

File: create-tnf/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@umijs/create-tnf",
```
Comment (xiaohuoni) [inline create-tnf/package.json]: 包名可以直接用 create-tnf ，更方便使用

Comment (xiaohuoni) [reply create-tnf/package.json]: https://www.npmjs.com/package/create-tnf

Comment (QDyanbing) [reply create-tnf/package.json]: 👌等 @sorrycc 提交后我再调整下

File: create-tnf/src/cli.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import assert from 'assert';
```
Comment (QDyanbing) [inline create-tnf/src/cli.ts]: 这个没用到

**Pull request merged**